.github/workflows/devconfig-validator.yml

```diff
@@ -0,0 +1,219 @@
+name: DevConfig Validator
```

Comment:
Should this workflow be skipped if the target is anything other then development or the V3 development? What I'm thinking is the PR is an incremental work for a feature branch a dev config is not required.